ansible/roles/beegfs/tasks/client.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Install packages for BeeGFS client
+  package:
+    name: "{{ item }}"
+    state: present
+  become: True
+  with_items:
+  - "beegfs-client"
+  - "beegfs-helperd"
+  - "beegfs-utils"
+
+- name: Ensure the BeeGFS client services are not running
+  service:
+    name: "{{ item }}"
+    state: "stopped"
+  become: True
+  with_items:
+  - "beegfs-helperd"
+  - "beegfs-client"
+
+- name: Define client kernel build for IB
+  block:
+  - name: Define client kernel build for IB
+    lineinfile: 
+      path: "/etc/beegfs/beegfs-client-autobuild.conf"
+      regexp: "^buildArgs="
+      line: "buildArgs=-j8 BEEGFS_OPENTK_IBVERBS=1"
+#     line: "buildArgs=-j8 BEEGFS_OPENTK_IBVERBS=1 OFED_INCLUDE_PATH=/usr/src/ofa_kernel/default/include"
+  - name: Detect the version of the running kernel
+    command: "uname -r"
+    register: kernel_version
+  - name: Ensure kernel development headers are present
+    package:
+      name: "kernel-devel-{{ kernel_version.stdout }}"
+      state: present
+  become: True
+  when: beegfs_rdma
+
+- name: Ensure gcc is installed
+  package:
+    name: "gcc"
+    state: present
+  become: True
+
+- name: Rebuild the BeeGFS client kernel module
+  command: /etc/init.d/beegfs-client rebuild
+  become: True
```

Comment:
Is this going to turn up changed on every run, I wonder if we can depend on the module being built, or something like that?